services/ops/MessageDebuggerAgent/README.md

```diff
@@ -0,0 +1,615 @@
+## Message Debugger Agent
+
+VOLTTRON agent messages are routed over the VOLTTRON message bus.  The Message Debugger Agent provides enhanced
+examination of this message stream's contents as an aid to debugging and troubleshooting agents and drivers.
+
+This feature is implemented to provide visibility into the ZeroMQ message bus.  The RabbitMQ message bus includes
+methods for message debugging by default in the `RabbitMQ management UI <https://www.rabbitmq.com/management.html>`_.
```

Comment:
RST link, should be [RabbitMQ management UI](https://www.rabbitmq.com/management.html)